crates/proto/src/generated/note.rs

```diff
@@ -31,11 +40,37 @@ pub struct NoteSyncRecord {
     #[prost(message, optional, tag = "2")]
     pub note_id: ::core::option::Option<super::digest::Digest>,
     #[prost(message, optional, tag = "3")]
-    pub sender: ::core::option::Option<super::account::AccountId>,
-    #[prost(fixed32, tag = "4")]
-    pub tag: u32,
-    #[prost(uint32, tag = "5")]
-    pub note_type: u32,
-    #[prost(message, optional, tag = "6")]
+    pub metadata: ::core::option::Option<NoteMetadata>,
+    #[prost(message, optional, tag = "4")]
     pub merkle_path: ::core::option::Option<super::merkle::MerklePath>,
 }
+#[derive(Eq, PartialOrd, Ord, Hash)]
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash, PartialOrd, Ord, ::prost::Enumeration)]
```

Comment:
I have yet to avoid these conflicting derives from happening without making something too custom on `builld.rs`.

Reply:
Turns out we did not need those custom attributes, I thought I had tested it but apparently not (probably was testing it along other modifications I had made).